src/openqaoa-pyquil/backends/gates_pyquil.py

```diff
@@ -6,35 +6,39 @@
 from openqaoa.qaoa_components.ansatz_constructor.rotationangle import RotationAngle
 import openqaoa.qaoa_components.ansatz_constructor.gates as gates_core
 
-class PyquilGateApplicator(gates_core.GateApplicator):
 
+class PyquilGateApplicator(gates_core.GateApplicator):
     PYQUIL_OQ_GATE_MAPPER = {
+        gates_core.X.__name__: gates.X,
         gates_core.RZ.__name__: gates.RZ,
         gates_core.RX.__name__: gates.RX,
         gates_core.RY.__name__: gates.RY,
         gates_core.CX.__name__: gates.CNOT,
         gates_core.CZ.__name__: gates.CZ,
         gates_core.CPHASE.__name__: gates.CPHASE,
-        gates_core.RiSWAP.__name__: gates.XY
+        gates_core.RiSWAP.__name__: gates.XY,
     }
 
-    library = 'pyquil'
+    library = "pyquil"
+
+    def create_quantum_circuit(self, n_qubits) -> Program:
+        """
+        Function which creates and empty circuit for the specific backend.
+        Needed for twirling but can be used inside parametric circuit too
+        instead of creating parametric_circuit = QuantumCircuit(self.qureg)
+        TODO this function doesn't require n_qubits but the one for qiskit does.
```

Comment:
this function doesn't require n_qubits but the one for qiskit does. Is there a way to make it more generic or we just pass an extra param?